src/components/EntityInfo/EntityMenu.vue

```diff
@@ -197,7 +197,7 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     this.emitConfirmDissolution()
   }
 
-  goToConsentContinuationOutFiling (): void {
+  goToConsentContinuationOutFiling ():void {
```

Comment:
I had this fix but some other PR fixed this before mine

Reply:
Put the fix back in, otherwise this commit will remove that space.

Reply:
Looks good.